frontend/src/app/containers/ExperimentPage.js

```diff
@@ -276,10 +276,11 @@ export class ExperimentDetail extends React.Component {
                 <h2 className="banner">
                   <span data-l10n-id="experimentPromoHeader" className="block">Ready for Takeoff?</span>
                 </h2>
-                <p data-l10n-id="experimentPromoSubheader">We're building next-generation features for Firefox. Install Test Pilot to try them!</p>
+                <p data-l10n-id="experimentPromoSubheader">We&apos;re building next-generation features for Firefox. Install Test Pilot to try them!</p>
```

Comment:
Never mind, I forgot that what counts is the data-l10n-id. Given that the best practice would be to leave the text empty, it doesn't look like a relevant change.